schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+        -   Google isn't enough if you don't know what to search for or how to recognize a useful answer when you find one
+        -   GG added "awareness" to the bottom of the Bloom hierarchy since it's the most realistic goal for a workshop like this
+    -   Exercise: classify three learning objectives by level
+
+1.  Novice/Competent Practitioner/Expert (0:45/10:45)
+    -   <./keynote/20-novice-competent-expert.key>
+    -   Mental models
+        -   Emphasize linkages: we rarely forget completely, but we lose the ability to retrieve
+    -   Mental models don't have to be right to be useful (ball-and-spring models in chemistry)
+    -   How the graph model of human cognition expains the difference between competent and experts
+        -   Experts have more connections
+    -   Concept maps as visual representations of mental models
+    -   Using concept maps:
+        -   To design lesson
+        -   To convey information to learners
+        -   For insight into their learning
+    -   Exercise: draw and compare concept maps for ggplot2
+
+1.  Coffee (0:15/11:00)
+
+1.  Teaching as Performance I (0:30/11:30)
+    -   <./keynote/30-teaching-as-performance-1.key>
+    -   Critique the Nederbragt videos
+        -   Bad: <https://youtu.be/bXxBeNkKmJE>
+        -   Good: <https://youtu.be/SkPmwe_WjeY>
+    -   Exercise: teach in pairs (no recording)
+    -   Build rubric for use in second teaching session
+
+1.  Multimedia Learning (0:30/12:00)
+    -   <./keynote/80-multimedia-learning.key>
+    -   Exercise: create a visual to explain what the course is about
+
+## Day 2
+
+1.  The Cognitive Craft (0:45/09:45)
+    -   <./keynote/40-cognitive-craft.key>
+    -   Revisit cognitive architecture diagram and introduce 7 +/- 2
+    -   Lesson "scenes" should fit into working memory
+        -   Use exercises/discussion/elaborative examples to keep new material there long enough to be transcribed
+    -   Cognitive load theory
+        -   Intrinsic
+        -   Extraneous
+        -   Germane
+    -   Exercise: outline a faded example
+
+1.  Formative Assessment (0:45/10:30)
+    -   <./keynote/50-formative-feedback.key>
+    -   Importance of feedback loop while teaching (for instructor as well as learners)
+    -   MCQ example
+    -   Dual purposes of formative assessments (feedback loop + reinforcement)
+    -   Different kinds of formative assessment
+    -   Exercise: create MCQ and explain misconceptions it diagnoses
+
+1.  Coffee (0:15/10:45)
+
+1.  Teaching as Performance II (0:30/11:15)
+    -   <./keynote/60-teaching-as-performance-2.key>
+    -   Use the previous day's rubric
+
+1.  Motivation (0:30/11:45)
+    -   <./keynote/70-motivation.key>
+    -   Three big motivators: self-efficacy, utility, community
+    -   Three big demotivators: unpredictability, unfairness, indifference
+    -   Zone of Proximal Development
+    -   Exercise: what demotivates people in programming classes and how do we prevent/fix it?
```

Comment:
if we can make an exercise that helps coders realize that the syntax they choose can be demotivating, it would be a win for the entire R community.

Reply:
I need $5000 to make this happen...